CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing Topics to the Show
+
+## Submitting a Conference
+Submit Upcoming Conferences to be talked about on the show using one our [conference issue templates](https://github.com/python-community-news/topics/issues/new/choose).
+
+> NOTE: All Conference announcements should have an issue made. Issues will be closed and not reported on if they fail to meet the criteria for a valid conference, announcement type, or are not approved but the hosts.
+
+### Types of Conferences
+We would love you to submit any conference that would be of interest to the Python Community.
+
+Here are some examples:
+
+1. A Conference _celebrating Python in a geographic region or technical discipline ([PyCon US](https://pycon.us), [PyGotham](https://pygotham.org), [PyData](https://pydata.org/upcoming-events/))
+2. The Conference **Celebrates a Community where Python is widely used** ([TableauCon](https://www.tableau.com/events/conference), [Open Data Science Conference](https://odsc.com))
+3. The Conference is a General Technology Focus with a **Python/Data Science specific track** ([All Things Open](https://www.allthingsopen.org))
+4. A community-run general technology conference ([!!Con](https://bangbangcon.com))
```

Comment:
I'd argue that the text here is too broad to be an example of what we'll cover. !!Con is a good example of one we will / have covered, but not all "general technology conferences" will fit this bill, and most that do would fall into (3) above.